### PR TITLE
qxw: init at 20190909

### PR DESCRIPTION
**Pull request by tckmn**
###### Motivation for this change

Adds a derivation for [the `qxw` program](https://www.quinapalus.com/qxw.html).

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by puzzlewolf — CHANGES_REQUESTED** [review]
LGTM, the binary starts fine.

Commit message should be `qxw: init at 20190909`, just like PR title :)

**Comment by tckmn** [comment]
@puzzlewolf whoops, sorry -- fixed that!

**Review by puzzlewolf — APPROVED** [review]
LGTM

File: pkgs/applications/editors/qxw/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  patchPhase = ''
+    sed -i 's/ `dpkg-buildflags[^`]*`//g;
+            /mkdir -p/d;
+            s/cp -a/install -D/;
+            s,/usr/games,/bin,' Makefile
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A program to help create and publish crosswords";
+    homepage    = https://www.quinapalus.com/qxw.html;
```
Comment (puzzlewolf) [inline pkgs/applications/editors/qxw/default.nix:26]:
```suggestion
    homepage    = "https://www.quinapalus.com/qxw.html";
```
NixOS/rfcs#45

**Review by puzzlewolf — CHANGES_REQUESTED** [review]
Just one nitpick :)

**Comment by tckmn** [comment]
@puzzlewolf slightly delayed lol but fixed that as well! (and updated version because there's been a new one in the meantime)

**Comment by ryantm** [comment]
Result of `nixpkgs-review pr 82175` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- qxw
</details>

**Review by ryantm — APPROVED** [review]

**Pull request merged**